examples/chart/teleport-cluster/templates/deployment.yaml

```diff
@@ -251,7 +251,7 @@ spec:
         secret:
           secretName: "license"
 {{- end }}
-{{- if (eq .Values.chartMode "gcp") }}
+{{- if and (.Values.gcp.credentialSecretName) (eq .Values.chartMode "gcp")  }}
       - name: gcp-credentials
         secret:
           secretName: {{ required "gcp.credentialSecretName is required in chart values" .Values.gcp.credentialSecretName }}
```

Comment:
The `{{ required }}` part of the logic can likely be dropped, even though this is how it looked before the most recent change.

Reply:
Sounds sensible @programmerq - I've removed the required field